### DISCUSSION TITLE
The default accessKey and secretKey counter with "unauthorized" error

### DISCUSSION THREAD

**Discussion by prchanda** [minio/minio#12906] [Q&A] [answered]
### Discussed in https://github.com/minio/minio/discussions/12674

<div type='discussions-op-text'>

I am facing the same issue in Windows 10 host. Here is the windows version I am using:

![image](https://user-images.githubusercontent.com/35064849/128606644-b26ca85c-586c-4f0c-8916-ef493da4a5ab.png)

The version of minio.exe is：
![image](https://user-images.githubusercontent.com/35064849/128606675-0f30878d-c47d-4760-9330-537fe3d15206.png)

Using the following command to start minio.exe:
![image](https://user-images.githubusercontent.com/35064849/128607004-5230e8d8-8a06-4ec6-9ba8-62a08a0c1072.png)

Opened the browser and accessed the url http://127.0.0.1:9080/login to login with custom user credentials. Getting 401 error:
![image](https://user-images.githubusercontent.com/35064849/128607091-93399949-160d-4a3f-96fb-74d2da5a0d43.png)
![image](https://user-images.githubusercontent.com/35064849/128607147-f34662ac-2fda-4091-9c0b-7f15703404cb.png)

</div>

**Comment by parthibd** [comment]
Same here.

**Comment by harshavardhana** [comment]
Moved to discussion can you provide more information by running `mc admin trace -v alias/` - make sure to configure mc

**Reply by parthibd** [reply]
I am trying to do that but the command has just hung.

**Reply by prchanda** [reply]
![image](https://user-images.githubusercontent.com/35064849/128634831-d395d98d-07a4-4b6c-ac07-e18443833f25.png)
Yes it's hung. (local is the alias set in config.json)
@harshavardhana 
I kept the trace running and tried to login in the minio server via browser with the creds, nothing moves still in the trace.
However, when trying with Java minio client, it connects fine with the minio server.
![image](https://user-images.githubusercontent.com/35064849/128634983-c06a38d9-bd84-43f4-bb16-be5865ef8815.png)
So I suspect some config issue with mc.exe CLI?

**Reply by harshavardhana** [reply]
> ![image](https://user-images.githubusercontent.com/35064849/128634831-d395d98d-07a4-4b6c-ac07-e18443833f25.png)
> Yes it's hung. (local is the alias set in config.json)
> @harshavardhana
> I kept the trace running and tried to login in the minio server via browser with the creds, nothing moves still in the trace.
> However, when trying with Java minio client, it connects fine with the minio server.
> ![image](https://user-images.githubusercontent.com/35064849/128634983-c06a38d9-bd84-43f4-bb16-be5865ef8815.png)
> So I suspect some config issue with mc.exe CLI?

This means that browser is not sending requests to MinIO. 

Can you try MINIO_SERVER_URL=http://127.0.0.1:9000 and see if it works ?

**Comment by harshavardhana** [comment, marked as answer]
This should be fixed by #12910
